src/observations/topocentric.jl

```diff
@@ -118,123 +31,152 @@ end
     # Remote file 
-    eop_iau2000a_rf = RemoteFiles.@RemoteFile(EOP_IAU2000A_RF, "https://datacenter.iers.org/data/csv/finals2000A.all.csv", file="EOP_IAU2000A.TXT", updates=:weekly)
+    eop_iau2000a_rf = RemoteFiles.@RemoteFile(
+        EOP_IAU2000A_RF, 
+        "https://datacenter.iers.org/data/csv/finals2000A.all.csv", 
+        file = "EOP_IAU2000A.TXT", 
+        dir = observations_path, 
+        updates = :weekly
+    )
     # Download remote file 
-    RemoteFiles.download(EOP_IAU2000A_RF, force=force)
+    RemoteFiles.download(EOP_IAU2000A_RF, force = force)
     # Read delimited file 
     eop, ~ = readdlm(RemoteFiles.path(eop_iau2000a_rf), ';'; header = true)
     # Parse eop 
     SatelliteToolbox._parse_iers_eop_iau_2000A(eop)
 end
 
 # Earth orientation parameters (eop) 1980
-eop_IAU1980 = get_eop_iau1980()
+const eop_IAU1980 = get_eop_iau1980()
 # Earth orientation parameters (eop) 2000
-eop_IAU2000A = get_eop_iau2000a()
+const eop_IAU2000A = get_eop_iau2000a()
+
+@doc raw"""
+    obs_pos_ECEF(observatory::ObservatoryMPC{T}) where {T <: AbstractFloat}
+    obs_pos_ECEF(x::RadecMPC{T}) where {T <: AbstractFloat}
+    obs_pos_ECEF(x::RadarJPL{T}) where {T <: AbstractFloat}
+
+Return the observer's geocentric `[x, y, z]` position vector in Earth-Centered Earth-Fixed (ECEF) reference frame.
+"""
+function obs_pos_ECEF(observatory::ObservatoryMPC{T}) where {T <: AbstractFloat}
+
+    # Make sure observatory has coordinates 
+    @assert !isunknown(observatory) "Cannot compute position for unknown observatory."
+    @assert hascoord(observatory) "Cannot compute position for observatory [$(observatory.code)] without coordinates"
+
+    # λ_deg: longitude [degrees east of Greenwich]
+    # u: distance from spin axis [km], u = ρ*cos(ϕ')
+    # v: height above equatorial plane [km], v = ρ*sin(ϕ'),
+    # where ϕ' is the geocentric latitude and ρ is the geocentric distance in km
+
+    # Cilindrical components of Earth-Centered Earth-Fixed position of observer
+    λ_deg = observatory.long     # deg 
+    u = observatory.cos * RE     # km 
+    v = observatory.sin * RE     # km 
+
+    # Cartesian components of Earth-Centered Earth-Fixed position of observer
+    λ_rad = deg2rad(λ_deg)       # rad
+    x_gc = u * cos(λ_rad)        # km
+    y_gc = u * sin(λ_rad)        # km
+    z_gc = v                     # km
+
+    # Earth-Centered Earth-Fixed position position of observer 
+    pos_ECEF = [x_gc, y_gc, z_gc] # km
+
+    return pos_ECEF
+end
+
+obs_pos_ECEF(x::RadecMPC{T}) where {T <: AbstractFloat} = obs_pos_ECEF(x.observatory)
+obs_pos_ECEF(x::RadarJPL{T}) where {T <: AbstractFloat} = obs_pos_ECEF(x.rcvr)
+
+# isless overloads workaround to make rECEFtoECI work with TaylorSeries
+isless(a::Union{Taylor1,TaylorN}, b::Union{Taylor1,TaylorN}) = isless(constant_term(a), constant_term(b))
+isless(a::Real, b::Union{Taylor1,TaylorN}) = isless(a, constant_term(b))
+isless(a::Union{Taylor1,TaylorN}, b::Real) = isless(constant_term(a), b)
```

Comment:
Two comments:
- this is type piracy :-)
- a more general approach of this is already implemented in TS v0.15

So I think this is not needed... (included the related `import Base: isless`)

Reply:
I actually added this a while ago, way before TS v0.15 was released (probably should've opened an issue back then), but now that `isless` has been implemented for `TaylorSeries` polynomials, I agree this is not needed anymore. In other words, it wasn't type piracy until about two weeks ago 😅